### PR TITLE
Do not use pager for git attributes check

### PR DESCRIPTION
**Pull request by tsauerwein**
See https://github.com/camptocamp/ngeo/commit/7d4c2ace43172bea4f994e05c074f3396638adab#commitcomment-17929803

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -167,7 +167,7 @@ lint: .build/eslint.timestamp
 
 .PHONY: git-attributes
 git-attributes:
-	git diff --check HEAD~40
+	git --no-pager diff --check HEAD~40
```
Comment (tsauerwein) [inline Makefile]: No, before.

Comment (sbrunner) [inline Makefile]: Effectively, thanks :-)

**Comment by sbrunner** [comment]
Looks good, merge when Travis is happy :-)

**Comment by tsauerwein** [comment]
Thanks for the review

**Pull request merged**